testing/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ if (draco_enabled)
 endif ()
 
 if (occt_enabled)
-  f3d_add_test(TestOCCT f3d.stp --cells=0) # A simple OCCT test that work whether coloring is turned on or not
+  f3d_add_test(TestOCCT f3d.stp --scalars=f3d_reserved) # A simple OCCT test that work whether coloring is turned on or not
```

Comment:
isn't `--scalars` enough?

Reply:
--scalars turns it on, no way to turn it off, see https://github.com/f3d-app/f3d/issues/837